### ISSUE TITLE
add feedback link on api pages

### ISSUE THREAD

**Issue by gbinal** [USDA/USDA-APIs#12]
For each of the USDA APIs, add a `Feedback` link in the sidebar (where it's more visible) to [this issue tracker](https://github.com/USDA/USDA-APIs/issues) so that it's more prominent and accessible.

**Comment by adelevie** [comment]
I agree. As a developer, this will help me help you to improve the API.
